.github/workflows/code-quality.yaml

```diff
@@ -17,12 +17,13 @@ jobs:
         with:
           working_dir: src
           php_version: "7.4"
+          php_extensions: zip
 
       - name: Run PHPCS
         run: cd src && composer run lint
 
       - name: Run PHPUnit
-        run: cd src && composer run phpunit
+        run: cd src && composer run phpunit -- --exclude-group selenium_test
```

Comment:
In the follow-up PR, I'm thinking we can have a separate action to run tests (both selenium and non-selenium). The matrix strategy we use [here](https://github.com/orgnc/platform/blob/c880cf2da990b05b8fd1472eb8b47f35d640a63e/.github/workflows/pr-tests.yaml#L110) is probably overkill, but I think separating Code Quality from Tests is probably a good idea. What do you think @sergunich?